### PR TITLE
VIA Support: Doro 67

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Since the Regular PCB is basically just the RGB PCB sans the RGB, I also added the same changes there. That way, the same VIA json file can account for both. 

I reason that despite seeing that RGB keycodes are available in VIA, if you have the regular PCB you should know not to add them. (Coincidentally the default keymap for regular had an RGB keycode in it haha)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Wilba6582** [comment]
There will eventually need to be two VIA definitions to control the RGB of the RGB PCB, so might as well assign them separate VID/PID now and make the VIA definition names match the firmware names.

**Comment by Wilba6582** [comment]
Also, why not add the multi PCB at this time?

File: keyboards/doro67/regular/config.h [hunk 1]
```diff
@@ -3,8 +3,8 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x0000
+#define VENDOR_ID       0x4250 // BP for Backprop
+#define PRODUCT_ID      0x5247 // RG for Regular
```
Comment (fauxpark) [inline keyboards/doro67/regular/config.h]:
```suggestion
#define PRODUCT_ID      0x5243 // RE for Regular
```
?

File: keyboards/doro67/rgb/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    QMKBEST = SAFE_RANGE,
+    QMKURL,
+};
+
```
Comment (fauxpark) [inline keyboards/doro67/rgb/keymaps/via/keymap.c]:
```suggestion
```

File: keyboards/doro67/regular/config.h (hunk 1 above)
Comment (noroadsleft) [reply keyboards/doro67/regular/config.h]:
```suggestion
#define PRODUCT_ID      0x5245 // RE for Regular
```

E is `45`, not `43`.

Comment (fauxpark) [reply keyboards/doro67/regular/config.h]: 🤦‍♂

**Comment by mechmerlin** [comment]
> Also, why not add the multi PCB at this time?

There was someone on stream following along to do the multi. However I do not see a PR from him. I guess I'll put it in for him.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**